keyboards/southpole/.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This file should be removed from the pull request.

Reply:
And honestly, it shouldn't be needed.  the default file should associate all h files with c.

Reply:
omg sorry about that, I didn't realize this file got added in again...